Tests/MapboxMapsTests/Snapshot/SnapshotterTests.swift

```diff
@@ -28,19 +28,27 @@ final class SnapshotterTests: XCTestCase {
     }
 
     // Test snapshot start invokes mockMapSnapshotter startStub
-    func testSnapshotterStart() {
+    func testSnapshotterStartInvocation() {
```

Comment:
Could there be tests for the internals of `Snapshotter.start(overlayHandler:completion:)`? There seem to be quite a lot going on - I wonder if we can mock `MapSnapshot` to achieve this.